ksql-engine/src/main/java/io/confluent/ksql/function/udf/list/Sublist.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf.list;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+import java.util.List;
+
+@UdfDescription(name = "sublist", description = "sublist of a generic list")
+public class Sublist {
```

Comment:
Likewise, this is `subArray`. 

We may also want to expose this in a more Postgres way ;)  e.g. 

```sql
SELECT MY_ARRAY[4:6] FROM BLAH;
```

Would pick a sub list from elements 4 -> 6.

Under the hood this could still us the UDF...

Reply:
Totally agree about this - I changed it to `slice` and later we should introduce `[x:y]` syntax for this